### DISCUSSION TITLE
Button Toggler text size changing

### DISCUSSION THREAD

**Discussion by manimovassagh** [quasarframework/quasar#13957] [General - Components / Directives / etc]
Hi, 
is there any way to change QBtnToggle text size.
I have custom font installed and now the size is always 10 px and i want to change it to 14px.
Thanks

**Comment by MilosPaunovic** [comment]
Simple CSS: [https://codepen.io/milospaunovic/pen/abYBBQj](https://codepen.io/milospaunovic/pen/abYBBQj)

**Reply by manimovassagh** [reply]
Thanks a lot . how do you find that you have to use this class?

**Reply by manimovassagh** [reply]
.q-btn-group *  this one i mean

**Reply by MilosPaunovic** [reply]
By using browser element inspector tool.

**Reply by manimovassagh** [reply]
Thanks alot sorry for too much question but is there a way to have spacing between buttons all they have to always stick together?
i want to make a margin or gutter between them.
Again Thanks a lot really appreciate That

**Reply by MilosPaunovic** [reply]
I guess you could find the right class and apply margin property to it, but why not use normal buttons instead?

**Reply by manimovassagh** [reply]
sorry for late answer. i was in vacation and also had corona for a week. because i want to have some toggling and i want to use this function .
